README.md

```diff
@@ -166,12 +166,11 @@ Anchors are great for jumping to information farther down a page.
 
 #### Buttons vs.Links vs. Anchors
 ##### The semantic question of link vs button:
- * A button invokes an action: Save, Cancel, Exterminate.
-  * Activated with the `Space` or `Enter` key. Screen reader users may assume this is a form submission and be slightly more trepidacious about activating it than a link.
-  * Identified in screen readers as a button, and included in the list of form controls.
- * A link takes you somewhere (navigates).
-  * Activated with the `Enter` key.
-  * Identified in screen readers as a link, and included in the list of page links.
+| **Buttons** | **Links** |
+|-|-|
+| A button invokes an action: Save, Cancel, Exterminate. | A link takes you somewhere (navigates). |
+| Activated with the Space or Enter key. Screen reader users may assume this is a form submission and be slightly more trepidacious about activating it than a link. | Activated with the Enter key. |
+| Identified in screen readers as a button, and included in the list of form controls. | Identified in screen readers as a link, and included in the list of page links. |
 
 ##### Considerations when choosing an element:
  * If you style a link to look like a button, a keyboard user will not know to use the `Enter` key to click on it.
```

Comment:
You can remove this item, it's related to the form submission bit above.

Reply:
Done!